microservices/services/query-metric/service/src/main/java/datawave/microservice/querymetric/handler/QueryMetricCombiner.java

```diff
@@ -270,4 +279,9 @@ protected boolean isChanged(String updated, String stored) {
             return false;
         }
     }
+    
+    // check to see if we should be adding the updated subplans to the stored subplans. If false, set them instead of adding
+    protected boolean shouldAddSubplan(Map<String,RangeCounts> updated, Map<String,RangeCounts> stored) {
+        return (updated != null && stored != null) && (updated.size() == 1);
```

Comment:
Is size == 1 really the condition that you want to check here.  This seems odd.

Reply:
Improved the combining logic in the latest commit